templates/vue/src/components/node-modal/ReviewForm.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="review-form">
+    <b-form-group label="Reviewer's Comment">
+      <rich-text-form
+        id="node-description"
+        v-model="comment"
+        placeholder="Enter a comment"
+      />
+    </b-form-group>
+    <div>
+      <b-button size="sm" variant="light" @click="close">
+        Cancel
+      </b-button>
+      <b-button size="sm" variant="danger" @click="handleReject">
+        Reject
+      </b-button>
+      <b-button size="sm" variant="primary" @click="handleAccept">
+        Accept and Add
+      </b-button>
+    </div>
+  </div>
+</template>
+
+<script>
+import moment from "moment-timezone"
+import { getCurrentUser } from "@/services/wp"
+import RichTextForm from "./content-form/RichTextForm"
+
+export default {
+  components: {
+    RichTextForm,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      comment: "",
+    }
+  },
+  mounted() {
+    moment.tz.setDefault("America/Vancouver")
+  },
+  methods: {
+    handleReject() {
+      this.node.status = "draft"

```

Comment:
This line could be safely removed, I believe.

Reply:
Yep.